include/swift/AST/DiagnosticsFrontend.def

```diff
@@ -190,6 +190,9 @@ ERROR(error_index_failed_status_check,none,
 ERROR(error_index_inputs_more_than_outputs,none,
   "index output filenames do not match input source files", ())
 
+ERROR(error_too_many_supplementary_outputs,none,
+"too many supplementary outputs '%0'", (StringRef))
```

Comment:
This should probably just be `too many %0 arguments` or perhaps `too many %0 arguments (expected %1, got %2)`.

Reply:
Very nice! Fixed.